cst438_gradebook/src/main/java/com/cst438/controllers/GradeBookController.java

```diff
@@ -47,8 +49,7 @@ public class GradeBookController {
 	@GetMapping("/gradebook")
```

Comment:
Viewing gradbook code looks good, grades can be searched by email for student class assignment.